/scara_motor:
  ros__parameters:
    
    offset:
      joint1: 0.0
      joint2: -1.28
      joint3: -2.71
      joint4: 1.571
      joint5: 0.0
      joint6: 0.0
      joint7: -1.571

    ratio:
      joint1: 973.4
      joint2: 1.0
      joint3: 1.0
      joint4: 20.0
      joint5: 20.0
      joint6: 20.0
      joint7: 108.0